R/broom_wrapper.R

```diff
@@ -433,7 +433,12 @@ prediction_binary <- function(df, threshold = 0.5, ...){
   }
 
   # get actual value column name from model formula
-  actual_col <- all.vars(first_model$formula)[[1]]
+  actual_col <- if(!is.null(first_model$formula)) {
```

Comment:
I think we'd better check class of model.
I think $formula did not exist in lm etc. They have $terms instead.
Please take a look at what I did in model_stats().